package.json

```diff
@@ -38,6 +38,7 @@
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
     "semantic-release": "^6.3.2",
+    "simple-statistics": "^4.0.0",
```

Comment:
Nothing for now! But was thinking of using it for summary stats of the test data.